src/metabase/models/setting.clj

```diff
@@ -55,7 +55,8 @@
 (u/strict-extend (class Setting)
   models/IModel
   (merge models/IModelDefaults
-         {:types (constantly {:value :encrypted-text})}))
+         {:types (constantly {:value :encrypted-text})
+          :primary-key (constantly :key)}))
```

Comment:
Not related to this PR, but I think it's just missing. 
Is it ok to put it here?

Reply:
yeah 👍